src/controller/java/src/chip/devicecontroller/model/InvokeElement.java

```diff
@@ -81,10 +81,17 @@ public byte[] getTlvByteArray() {
   }
 
   @Nullable
-  public JSONObject getJson() {
+  public JSONObject getJsonObject() {
     return json;
   }
 
+  @Nullable
+  public string getJsonString() {
```

Comment:
Why return string, not String? json.toString() return String class

Reply:
typo, fixed, thanks